client/src/actions/profile.js

```diff
@@ -162,3 +162,84 @@ export const getAllPetProfiles = () => async (dispatch) => {
     });
   }
 };
+
```

Comment:
Please delete all the empty lines. Or if there is any special meanings for this? Just wonder...